static/app/components/events/eventReplay/replayClipPreview.tsx

```diff
@@ -0,0 +1,333 @@
+import ExternalLink from 'sentry/components/links/externalLink';
+import Link from 'sentry/components/links/link';
+import List from 'sentry/components/list';
+import ListItem from 'sentry/components/list/listItem';
+import Placeholder from 'sentry/components/placeholder';
+import {Flex} from 'sentry/components/profiling/flex';
+import ReplayTimeline from 'sentry/components/replays/breadcrumbs/replayTimeline';
+import {PlayerScrubber} from 'sentry/components/replays/player/scrubber';
+import useScrubberMouseTracking from 'sentry/components/replays/player/useScrubberMouseTracking';
+import {
+  Provider as ReplayContextProvider,
+  useReplayContext,
+} from 'sentry/components/replays/replayContext';
+import {ReplayPlayPauseBar} from 'sentry/components/replays/replayController';
+import ReplayPlayer from 'sentry/components/replays/replayPlayer';
+import ReplayProcessingError from 'sentry/components/replays/replayProcessingError';
+import {formatTime} from 'sentry/components/replays/utils';
+import {IconContract, IconDelete, IconExpand} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import getRouteStringFromRoutes from 'sentry/utils/getRouteStringFromRoutes';
+import {TabKey} from 'sentry/utils/replays/hooks/useActiveReplayTab';
+import useReplayReader from 'sentry/utils/replays/hooks/useReplayReader';
+import RequestError from 'sentry/utils/requestError/requestError';
+import useRouteAnalyticsParams from 'sentry/utils/routeAnalytics/useRouteAnalyticsParams';
+import useOrganization from 'sentry/utils/useOrganization';
+import {useRoutes} from 'sentry/utils/useRoutes';
+import useFullscreen from 'sentry/utils/window/useFullscreen';
+import useIsFullscreen from 'sentry/utils/window/useIsFullscreen';
+import {normalizeUrl} from 'sentry/utils/withDomainRequired';
+import FluidHeight from 'sentry/views/replays/detail/layout/fluidHeight';
+import {ReplayRecord} from 'sentry/views/replays/types';
+
+type Props = {
+  eventTimestampMs: number;
+  orgSlug: string;
+  replaySlug: string;
+  focusTab?: TabKey;
+  fullReplayButtonProps?: Partial<ComponentProps<typeof LinkButton>>;
+};
+
+function getReplayAnalyticsStatus({
+  fetchError,
+  replayRecord,
+}: {
+  fetchError?: RequestError;
+  replayRecord?: ReplayRecord;
+}) {
+  if (fetchError) {
+    return 'error';
+  }
+
+  if (replayRecord?.is_archived) {
+    return 'archived';
+  }
+
+  if (replayRecord) {
+    return 'success';
+  }
+
+  return 'none';
+}
+
+function ReplayPreviewPlayer({
+  toggleFullscreen,
+  replayId,
+  fullReplayButtonProps,
+}: {
+  replayId: string;
+  toggleFullscreen: () => void;
+  fullReplayButtonProps?: Partial<ComponentProps<typeof LinkButton>>;
+}) {
+  const routes = useRoutes();
+  const organization = useOrganization();
+  const isFullscreen = useIsFullscreen();
+  const {
+    currentTime,
+    startTimeOffsetMs: startTime,
+    durationMs: duration,
```

Comment:
Do we need to rename these to avoid conflicts or something? i'd prefer we didn't

Reply:
Ah I missed this when bulk renaming the variable, no need for it